.azure-pipelines/pipelines.yml

```diff
@@ -409,9 +409,29 @@ stages:
               artifactName: windows.release
             condition: always()
 
-      - job: docker
+      - job: clang_cl
```

Comment:
do we want to enable the deflaking script in this pipeline as well?

See where we enabled it on Windows https://github.com/envoyproxy/envoy/pull/14102/files#diff-8ca122481a2ae193003b523bf5b7a9cfa0e914be502d483b6adc1d06668c724a

Reply:
+1